### ISSUE TITLE
Can't use this with Node v22

### ISSUE THREAD

**Issue by maximebonhomme** [kozakdenys/qr-code-styling#231]
We're using `wagmi` which uses this library under the hood.
It looks like this [new release](https://github.com/kozakdenys/qr-code-styling/commit/04210364c067086863e42867c49897bca77308b7) has broken our build since this package is now locked in Node v22.

```bash
 ERR_PNPM_UNSUPPORTED_ENGINE  Unsupported environment (bad pnpm and/or Node.js version)

This error happened while installing the dependencies of wagmi@2.9.8
 at @wagmi/connectors@5.0.7
 at @metamask/sdk@0.20.3
 at @metamask/sdk-install-modal-web@0.20.2

Your Node version is incompatible with "qr-code-styling@1.7.1".

Expected version: 20.x
Got: v22.5.1

This is happening because the package's manifest has an engines.node field specified.
To fix this issue, install the required Node version.

```

**Comment by kozakdenys** [comment]
Thanks @maximebonhomme, I am looking into it 👀

**Comment by kozakdenys** [comment]
@maximebonhomme you can find fix in [v1.7.2](https://github.com/kozakdenys/qr-code-styling/releases/tag/v1.7.2)
